src/arith/int_set.cc

```diff
@@ -372,7 +387,27 @@ class IntervalSetEvaluator :
   }
 
   IntervalSet Eval(const PrimExpr& val) {
-    return this->VisitExpr(val);
+    IntervalSet result = this->VisitExpr(val);
```

Comment:
Ideally we do not want to add an additional case here, as it could recursive , perhaps need more thoughts

Reply:
This change is no longer needed.